napari/components/viewer_model.py

```diff
@@ -111,7 +111,7 @@ def palette(self, palette):
             return
 
         self._palette = palette
-        self.events.palette()
+        self.events.palette(palette=palette)
```

Comment:
I'm not sure about this fix, if you look at the function signature change in #1055 I made `_update_palette` now take an `event`. Isn't the actual fix to change https://github.com/napari/napari/blob/51cedac63cb166c2851f10a28a86461aca43f635/napari/_qt/qt_main_window.py#L89 and https://github.com/napari/napari/blob/master/napari/_qt/qt_main_window.py#L98-L100 to have the new signature?